lib/tasks/jasmine-rails_tasks.rake

```diff
@@ -2,10 +2,14 @@ namespace :spec do
 
   desc "run test with phantomjs"
   task :javascript => [:environment] do
-    require 'jasmine_rails/runner'
-    spec_filter = ENV['SPEC']
-    reporters = ENV.fetch('REPORTERS', 'console')
-    JasmineRails::Runner.run spec_filter, reporters
+    if Rails.env.test?
+      require 'jasmine_rails/runner'
+      spec_filter = ENV['SPEC']
+      reporters = ENV.fetch('REPORTERS', 'console')
+      JasmineRails::Runner.run spec_filter, reporters
+    else
+      system("RAILS_ENV=test bundle exec rake #{ARGV.join(' ')}")
```

Comment:
Hm, will this account for quoted values in ARGV? If that could happen. If one value was "foo bar", I think those quotes would be gone.

This might do the trick: `system("RAILS_ENV=test", "bundle", "exec", "rake", *ARGV)`